extensions/microsoft-authentication/src/AADHelper.ts

```diff
@@ -71,6 +72,15 @@ export interface ITokenResponse {
 	id_token?: string;
 }
 
+export interface IMicrosoftTokens {
+	accessToken: string;
+	idToken?: string;
+}
+
+export interface MicrosoftAuthenticationSession extends vscode.AuthenticationSession {
```

Comment:
thanks for creating the typings file! Sorry, I was unclear here, can you move this interface completely into that file? For other extensions to use the typings, they have to copy them over, for example the git extension has a `git.d.ts` file that I directly copy into a typings folder in the github pull request extension

Reply:
Gotcha, thanks for providing the example.